README.md

```diff
@@ -8,27 +8,25 @@ Crosswalk-app-tools is in very early stages of development, and not suitable for
 
 ### Preparation
 
-Debian testing is the only tested platform. Node.js, the debuild tool,
-the Crosswalk runtime, and git must be functional.
+Debian and Ubuntu is the only tested platform. Node.js, the debuild tool, the Crosswalk runtime, and git must be functional.
```

Comment:
"are the only", not "is the only". Plus it would be good to mention the versions that have been verified.